packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1717,6 +1754,45 @@ export class StripeHelper extends StripeHelperBase {
     }
   }
 
+  async fetchInvoicesForActiveSubscriptions(
+    uid: string,
+    status: Stripe.InvoiceListParams.Status,
+    earliestCreatedDate?: Date
+  ) {
+    const accountCustomer = await getAccountCustomerByUid(uid);
```

Comment:
This method should take the customer ID as an argument instead of relying on us still having the accountCustomer link in the database (which is the first thing we delete).